docs/features/logging.rst

```diff
@@ -0,0 +1,189 @@
+Logging
+=======
+
+Description
+-----------
+
+Django-Socio-GRPC has a built-in way to handle the logging of the errors of your services.
+    
+Usage
+-----
+
+Django-Socio-GRPC uses a system of logging based on the one used by Django.
+
+Here's a short reminder on the system it uses.
+
+It uses 4 main objects for it.
+    - Loggers
+    - Handlers
+    - Filters
+    - Formatters
+
+=======
+Loggers
+=======
+
+A logger is the entry point into the logging system. Each logger is a named bucket to which messages can be written for processing.
+
+A logger is configured to have a log level. This log level describes the severity of the messages that the logger will handle. Python defines the following log levels:
+
+    - DEBUG: Low level system information for debugging purposes
+    - INFO: General system information
+    - WARNING: Information describing a minor problem that has occurred.
+    - ERROR: Information describing a major problem that has occurred.
+    - CRITICAL: Information describing a critical problem that has occurred.
+
+========
+Handlers
+========
+
+The handler is the engine that determines what happens to each message in a logger. It describes a particular logging behavior, such as writing a message to the screen, to a file, or to a network socket.
+
+Like loggers, handlers also have a log level. If the log level of a log record doesn’t meet or exceed the level of the handler, the handler will ignore the message.
+
+=======
+Filters
+=======
+
+A filter is used to provide additional control over which log records are passed from logger to handler.
+
+==========
+Formatters
+==========
+
+Ultimately, a log record needs to be rendered as text. Formatters describe the exact format of that text. A formatter usually consists of a Python formatting string containing LogRecord attributes; however, you can also write custom formatters to implement specific formatting behavior.
+
+
+Here's an example of a Logging setup that uses all four of these objects.
+
+.. code-block:: python
```

Comment:
this example may not implement solution or should not be in upur doc as junior dev can have tendancies to copy/paste example directly in code and this example does not just work out of the box.
What your opinion on that @legau